packages/client/src/components/form/DocumentUploadfield/SimpleDocumentUploader.tsx

```diff
@@ -137,7 +137,9 @@ class SimpleDocumentUploaderComponent extends React.Component<
                 type:
                   allowedDocType.join() === 'image/png'
                     ? 'png'
-                    : allowedDocType.join()
+                    : allowedDocType
+                        .map((docTypeStr) => docTypeStr.split('/').pop())
+                        .join()
```

Comment:
This will also cover the case where we only have a single `allowedDocType` right? So do we still need to separately check for 'image/png' case?